src/main/java/org/apache/datasketches/tuple/Sketch.java

```diff
@@ -33,10 +33,10 @@
 
   protected static final byte PREAMBLE_LONGS = 1;
 
-  long[] keys_;
-  S[] summaries_;
-  long theta_;
-  boolean isEmpty_ = true;
+  //long[] hashArr_;
+  //S[] summaryArr_;
```

Comment:
You may have intended to remove these variables.

Reply:
Good catch. Thanks!